core/src/main/scala/org/apache/spark/rdd/ReliableCheckpointRDD.scala

```diff
@@ -83,8 +84,9 @@ private[spark] class ReliableCheckpointRDD[T: ClassTag](
    * Return the locations of the checkpoint file associated with the given partition.
    */
   protected override def getPreferredLocations(split: Partition): Seq[String] = {
-    val status = fs.getFileStatus(
-      new Path(checkpointPath, ReliableCheckpointRDD.checkpointFileName(split.index)))
+    val path = new Path(checkpointPath, ReliableCheckpointRDD.checkpointFileName(split.index))
```

Comment:
This is to determine the actual path to the part file. And if we plan to not parse the old format, then we can hardcode it to the new format, because if we keep it flexible - we will have to deal with figuring out what formatting was used or else we would be dealing with `FNF` exceptions.